src/lcm/input_processing/util.py

```diff
@@ -70,12 +70,21 @@ def get_variable_info(model: Model) -> pd.DataFrame:
         for var in variables
     ]
 
-    auxiliary_variables = _get_auxiliary_variables(
-        state_variables=info.query("is_state").index.tolist(),
+    enter_concurrent_valuation = _get_variables_that_enter_concurrent_valuation(
+        states_and_actions_names=list(variables),
         function_info=function_info,
         user_functions=model.functions,
     )
-    info["is_auxiliary"] = [var in auxiliary_variables for var in variables]
+    info["enters_concurrent_valuation"] = [
+        var in enter_concurrent_valuation for var in variables
+    ]
```

Comment:
```suggestion
    info["enters_concurrent_valuation"] = _indicator_enters_concurrent_valuation(
        states_and_actions_names=list(variables),
        function_info=function_info,
        user_functions=model.functions,
    )
```
No strong opinion about the naming, but the function should directly return what we need.

Same below.